### ISSUE TITLE
Prerendered map mask with the explored area

### ISSUE THREAD

**Issue by unchiujar** [unchiujar/Umbra#55]
Instead of redrawing everything on location change or map pan/zoon,  create a prerendered  bitmap to overlay over the maps. This can be a hybrid approach, some data is prerendered (previous explored area) some is drawn on the fly like the current explored area if the user has the screen on. Once the screen is of the newly explored area is merged into the prerendered data. 
It either needs to have a flag in the database saying the data has been rendered or have the rendered data deleted.
